export.html

```diff
@@ -0,0 +1,226 @@
+          display: none;
+        }
+        hr.mxPageBreak {
+          display: none;
+        }
+      }
+
+      @media screen {
+        table.mxPageSelector {
+          position: fixed;
+          right: 10px;
+          top: 10px;
+          font-family: Arial;
+          font-size: 10pt;
+          border: solid 1px darkgray;
+          background: white;
+          border-collapse: collapse;
+        }
+        table.mxPageSelector td {
+          border: solid 1px gray;
+          padding: 4px;
+        }
+        body.mxPage {
+          background: gray;
+        }
+      }
+    </style>
+
+    <script>
+      const mxLoadStylesheets = false;
+    </script>
+    <script src="vendor/drawio/src/main/webapp/js/app.min.js"></script>
+    <script>
+      Editor.initMath('vendor/drawio/src/main/webapp/math/MathJax.js');
+
+      // Exposed for Puppeteer
+      function render(input, pageIndex, format) {
+        const TRANSPARENT_COLORS = [
+          '',
+          'none',
+        ];
+        const TRANSPARENT_FORMATS = [
+          'png',
+          'svg',
+        ];
+
+        console.debug('Rendering page', pageIndex, 'of input', input, 'as', format);
+
+        function initializeGraph(body) {
+          console.debug('Initialising empty document');
+          document.body.innerHTML = '';
+          const container = document.createElement('div');
+          container.id = 'graph';
+          container.style.width = '100%';
+          container.style.height = '100%';
+          body.appendChild(container);
+
+          console.debug('Initialising empty graph');
+          const graph = new Graph(container);
+          graph.setEnabled(false);
+          graph.foldingEnabled = false;
+
+          return {container, graph};
+        }
+
+        function monkeypatchGraph(graph) {
+          console.debug('Monkey patching mxGraph to disable custom links');
+
+          const graphGetLinkForCell = graph.getLinkForCell;
+          graph.getLinkForCell = function(cell) {
+            const link = graphGetLinkForCell.apply(this, arguments);
+            if (link != null && this.isCustomLink(link)) {
+              link = null;
+            }
+
+            return link;
+          };
+
+          const cellRendererRedrawLabelShape = graph.cellRenderer.redrawLabelShape;
+          graph.cellRenderer.redrawLabelShape = function(shape) {
+            cellRendererRedrawLabelShape.apply(this, arguments);
+            if (shape.node != null) {
+              const links = shape.node.getElementsByTagName('a');
+
+              for (const i = 0; i < links.length; i++)
+              {
+                const href = links[i].getAttribute('href');
+                if (href != null && graph.isCustomLink(href)) {
+                  links[i].setAttribute('href', '#');
+                }
+              }
+            }
+          };
+
+          // Fits the number of background pages to the graph
+          if (false /* PDF specific */) {
```

Comment:
This should be removed for now. We'll add a `format` parameter here when we need it rather than always applying patches we don't need.